models/restaurants.go

```diff
@@ -1,27 +1,45 @@
 package models
 
+import (
+	"database/sql"
+	"fmt"
+)
+
+// Restaurant data struct
 type Restaurant struct {
-	ID   string `json:"id"`
-	Name string `json:"name"`
+	ID      int            `json:"id"`
+	Name    string         `json:"name"`
+	Address string         `json:"address"`
+	Number  sql.NullString `json:"number"`
```

Comment:
we'll eventually have to wrap this to handle Null case... I feel like we should just get rid of nullable column for now. Thoughts?